<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParsedWindow</class>
 <widget class="QWidget" name="ParsedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Универсальный парсер</string>
  </property>
  <widget class="QTableWidget" name="result_product_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>451</height>
    </rect>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="save_2_csv_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить в CSV</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
